doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,168 @@
+dotnet sln remove tests\Azure.<group>.<service>.Tests.csproj
+dotnet sln add tests\Azure.<group>.<service>.Tests.csproj
+```
 
-Your `autorest.md` file should now look like:
+**Learn more:** to understand more about the Azure SDK repo structure, see [Repo Structure](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md) in the `azure-sdk` repo.
 
-````md
-# `Azure.<group>.<service>` Code Generation
+### Generate client library
 
-Run `dotnet build /t:GenerateCode` to generate code.
+In this section, we'll create a generated API layer built on Azure Core.
 
-### AutoRest Configuration
-> see https://aka.ms/autorest
+From a PowerShell command prompt, navigate to the directory holding `Azure.<group>.<service>.csproj`. Run the following commands:
 
-``` yaml
-input-file:
-- <swagger-file-link>
-namespace: <namespace>
-public-clients: true
-data-plane: true
-security: AADToken
-security-scopes: <AAD-token-scope>
 ```
-````
+sdk\<your-service-name>\Azure.<group>.<service>\src> dotnet build /t:GenerateCode
+```
 
-**Learn more:** see the [README.md](https://github.com/Azure/autorest.csharp/blob/feature/v3/readme.md) to learn more about Azure SDK Code Geneneration.
+After you run the GenerateCode command, you should find a **Generated** folder in your project. Inside the Generated folder you'll find the ServiceClient and ServiceClientOptions which you can use to interact with the service.
 
-#### 2. Run the Generate Command
+### Export public API
 
-From a PowerShell command prompt, navigate to the directory holding `Azure.<group>.<service>.sln`. Run the following commands:
+If you make public API changes or additions, the `eng\scripts\Export-API.ps1` script has to be run to update public API listings. This generates a file in the library's directory similar to the example found in [sdk\template-dpg\Azure.ServiceTemplate.Template\api\Azure.ServiceTemplate.Template.netstandard2.0.cs](https://github.com/Azure/azure-sdk-for-net/blob/bb0fbccfc33dd27d1ec6f0870022824d47181e61/sdk/template-dpg/Azure.ServiceTemplate.Template/api/Azure.ServiceTemplate.Template.netstandard2.0.cs).
 
+e.g. Running the script for a project in `sdk\deviceupdate` would look like this: 
 ```
-sdk\<your-service-name>\Azure.<group>.<service>\src> dotnet msbuild /t:GenerateCode
+eng\scripts\Export-API.ps1 deviceupdate
 ```
 
-#### 3. Explore the Generated Code
-
-After you've run the `GenerateCode` command above, you should find a `Generated` folder in your `Azure.<group>.<service>` project.
-
-If you're interested, you can look through the swagger file to see how the autorest.csharp extension translated the REST API definition into C# code. This could help you later if you decide to make adjustments to your API design in the swagger file directly, so they can be used for all Tier 1 languages.
-
-**Learn more**: see the [OpenAPI Swagger Documentation](https://swagger.io/docs/) to learn more about swagger file formats.
+## Add package ship requirements
```

Comment:
It would be good to start this section with a statement about what's required by the Azure SDK Guidelines to ship a library.  You can link to those sections of the guidelines as well.

e.g.
In order to ship an Azure SDK library, there are additional things that must be added to the package, including Tests, ... as described in https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-repository

Reply:
updated